tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/date_and_time.py

```diff
@@ -21,3 +29,59 @@ def __init__(self, marionette):
     def select_time_format(self, time_format):
         self.marionette.find_element(*self._24h_selector_locator).tap()
         self.select(time_format)
+
+    def set_automatic_time_update(self, state):
+        """
+        toggles 'Set Time Automatically' flag
+        """
+
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(
+                expected.element_present(*self._autotime_enabled_locator))))
+
+        if not state:
+            if self.is_autotime_enabled is True:
```

Comment:
But this method has to both enable and disable the switch depending on the 'state' value, and since I need to enable/disable it on the same script, it has to know whether it is already in the state that I want to put it in.  In the end, what matters is that it is in the correct state.  If I can't put it in the date_and_time.py, should I move this method out to the test script?